podium-gateway/src/main/java/nl/thehyve/podium/service/RequestFileService.java

```diff
@@ -194,6 +194,12 @@ public InputStreamResource getFileResource(UUID requestUuid, UUID fileUuid) thro
         return representations;
     }
 
+    public List<RequestFileRepresentation> getSortedFilesForRequest(UUID requestUUID){
```

Comment:
Why not have them always sorted by the database, i.e., changing this line: https://github.com/thehyve/podium/blob/8fe2aea43b102dfe409d50cb4059f2bd9e919c10/podium-gateway/src/main/java/nl/thehyve/podium/service/RequestFileService.java#L186

Reply:
Either by specifying a function `List<RequestFile> findDistinctByRequestAndDeletedFalseOrderByCreatedDate(Request request);` in the RequestFileRepository, or by adding a parameter `Pageable pageable`, and then let the frond end pass `createdDate` as order parameter.